test/tasks.spec.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env mocha */
+const expect = require('chai').expect
+const lib = require('../lib/file-helper')
+
+describe('tasks', () => {
+  describe('when running copy-to-destination', () => {
+    // Check to have all the components in dist and nothing else
+    it('should copy all components to dist', () => {
+      expect(lib.DistComponentList).to.deep.equal(lib.SrcComponentList)
+    })
+
+    // Check to have all the components (plus all, globals, icons) in packages and nothing else
+    it('should copy all components plus "all", "globals" and "icons" to packages', () => {
+      let expectedPackagesComponentList = lib.SrcComponentList.slice()
+      expectedPackagesComponentList.push('all', 'globals', 'icons')
+      expectedPackagesComponentList.sort()
```

Comment:
Could we add a comment here to say that there's no need to sort the files for `to.have.same.members` but it makes debugging failing tests a lot easier when the files are sorted?

Reply:
Sure. Comments added.